bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/zk/ZKMetadataDriverBase.java

```diff
@@ -289,6 +297,57 @@ public void processResult(int rc, String path, Object ctx, String name) {
         return createResult;
     }
 
+    private String getLedgerForReplicasMigrationPath(long ledgerId) {
+        return String.format("%s/%s", replicasMigrationPath, ledgerId);
+    }
+
+    public CompletableFuture<Void> submitToMigrateReplicas(long ledgerId, String bookieIds) {
+        CompletableFuture<Void> createResult = new CompletableFuture<>();
+        zk.create(getLedgerForReplicasMigrationPath(ledgerId), bookieIds.getBytes(StandardCharsets.UTF_8), acls,
```

Comment:
We'd better use `zk.multi()` operation to reduce the ZK RPC times.

Reply:
Fixed. 
PTAL,thanks!  @hangc0276